site/src/patterns/chat.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Chat
+description: The chat pattern in the Atlas Design System
+template: standard
+---
+
+# Chat Pattern
+
+The message conversation would be used in the following manner.
```

Comment:
I think attempting to write out a brief explanation would be helpful. Remember, human's might actually read this!

```suggestion
The following markup utilizes various components and atomics to create a chat experience. As you can see from the example below, we create a grid container and place several `.message` components within. In each message we make use of `.persona` to display a user's information, and a `popover` to house the overflow menu and its contents.
```

Reply:
updated description